tests/CommunityToolkit.Mvvm.SourceGenerators.Roslyn431.UnitTests/CommunityToolkit.Mvvm.SourceGenerators.Roslyn431.UnitTests.csproj

```diff
@@ -14,7 +14,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\CommunityToolkit.Mvvm\CommunityToolkit.Mvvm.csproj" />
-    <ProjectReference Include="..\..\CommunityToolkit.Mvvm.SourceGenerators.Roslyn430\CommunityToolkit.Mvvm.SourceGenerators.Roslyn430.csproj" />
+    <ProjectReference Include="..\..\CommunityToolkit.Mvvm.SourceGenerators.Roslyn431\CommunityToolkit.Mvvm.SourceGenerators.Roslyn431.csproj" />
```

Comment:
Think there's a missing link to this somewhere in the build?

> C:\hostedtoolcache\windows\dotnet\sdk\7.0.100-rc.1.22431.12\NuGet.targets(396,5): error MSB3202: The project file "D:\a\1\s\tests\CommunityToolkit.Mvvm.Roslyn431.UnitTests\CommunityToolkit.Mvvm.Roslyn431.UnitTests.csproj" was not found. [D:\a\1\s\dotnet Community Toolkit.sln]

Reply:
@Sergio0694 The error says **Project file not found**. I don't think it is related to the PR #450. Maybe you forgot to commit this `CommunityToolkit.Mvvm.Roslyn431.UnitTests` renamed file?

Reply:
@Sergio0694 The error says **Project file not found**. I don't think it is related to the PR #450. Maybe you forgot to commit this `CommunityToolkit.Mvvm.Roslyn431.UnitTests` renamed file?

Reply:
Fixed in https://github.com/CommunityToolkit/dotnet/pull/462/commits/f3977549a6780ff5d07e8c68cce9ed310efb0fe2 👍